resources/api-docs/1.0/relations.md

```diff
@@ -32,6 +32,7 @@ You can get a list of all the relations of an entity by using the following endp
             "attitude": 22,
             "visibility_id": 1,
             "is_star": false,
```

Comment:
We can remove `is_star` from the docs, because we want people to use `is_pinned` instead